### ISSUE TITLE
Update missing HER promo high res images

### ISSUE THREAD

**Issue by luceleaftea** [the-fab-cube/flesh-and-blood-cards#174]
Currently missing:

- [x] HER054 - Waiting on LSS to upload it
- [x] HER055 - Waiting on LSS to upload it
- [x] HER056 - Waiting on LSS to upload it
- [x] HER057 - Waiting on LSS to upload it
- [ ] HER064 - Waiting on LSS to upload it
- [x] HER074 - Waiting on LSS to upload it

**Comment by luceleaftea** [comment]
LSS has uploaded HER054-HER057, but the URL that should have the high-res images seem to have the 450-width images. I will use the high-res image link and notify them to see if that was a mistake or on purpose!

**Comment by luceleaftea** [comment]
Moved last image into its own issue at #294, closing this issue
